hybrasyl/Objects/User.cs

```diff
@@ -1260,163 +1188,74 @@ public void SendVisibleGold(Gold gold)
         internal void UseSkill(byte slot)
         {
             var castable = SkillBook[slot];
-            if (canCast(castable))
+            if (UseCastable(castable))
             {
-                bool spellCast = false;
-                foreach (var tar in PossibleTargets(castable))
-                {
-                    bool canAttack = !(tar is Merchant || (tar is User && (!(tar as User).Status.HasFlag(PlayerCondition.Pvp))));
-                    if (canAttack)
-                    {
-                        Attack(castable, tar);
-                        spellCast = true;
-                    }
-                }
-                if (spellCast) CastingCost(castable);
-
                 Client.Enqueue(new ServerPacketStructures.Cooldown()
                 {
                     Length = (uint)castable.Cooldown,
                     Pane = 1,
                     Slot = slot
-                }.Packet()); ;
+                }.Packet());
             }
+            else
+                SendSystemMessage("It didn't work.");

```

Comment:
"Failed." is the proper message

Reply:
OK - `Failed.` handles a specific case in usda and this is more broad, but, fine.